_cucumber/Dockerfile

```diff
@@ -4,8 +4,16 @@ RUN groupadd -g 1000 testing
 RUN useradd -g testing -m -s /bin/bash -u 1000 testing
 USER testing
 WORKDIR /home/testing
+
+USER root
```

Comment:
@RubyGemTSL Is there a reason we need to do this as root? It would be good if we could keep things as the `testing` user in order to avoid any issues with file permissions when running from Jenkins.